### ISSUE TITLE
[BUG] Tailscale up command quotes break the command

### ISSUE THREAD

**Issue by Daxcor69** [artis3n/ansible-role-tailscale#203]
**Describe the bug**
my tailscale won't authenticate with auth key or api key.   the error message doesn't make sense, You don't pass the api key, so why would it be not correct.  I tried to send on any way. 

**To Reproduce**
```
---
- name: Servers
  hosts: gateway
  gather_facts: yes
  become: no

  tasks:
    - name: Configure Sysctl
      sysctl:
        name: net.ipv4.ip_forward
        value: 1
        state: present
        ignoreerrors: true
        sysctl_set: true

    - name: Configure Tailscale
      include_role:
        name: artis3n.tailscale

      vars:
        tailscale_args: "--accept-routes=true --advertise-routes=10.0.1.0/24"
        # Pulled from the env vars on the host running Ansible
        tailscale_authkey: "blah blah"
```

**Expected behavior**
Tailscale would show as active.

**Screenshots**

**Target (please complete the following information):**
 - OS:  Ubuntu 20.04 minimal
 - Ansible version: ansible [core 2.12.5]
 - `artis3n.tailscale` version:   latest
 

** outputs:**

TASK [artis3n.tailscale : Get Existing State Checksum] ***************************************************************************************************************************************************
task path: /home/bd/.ansible/roles/artis3n.tailscale/tasks/lib/up-state.yml:2
fatal: [10.0.1.2]: FAILED! => {"changed": false, "msg": "file not found: /home/bd/.artis3n-tailscale/.idempotency/state"}
...ignoring

TASK [artis3n.tailscale : Install | Report redacted failure from "tailscale up"] *************************************************************************************************************************
task path: /home/bd/.ansible/roles/artis3n.tailscale/tasks/install.yml:101
fatal: [10.0.1.2]: FAILED! => {"changed": false, "msg": ["backend error: invalid key: invalid API token"]}

**Comment by Daxcor69** [comment]
I also tried via env and with hardcoding the keys in the playbook

**Comment by artis3n** [comment]
Thanks for the example playbook. I'll take a look at reproducing this tomorrow but from the error message, it appears the authkey is not valid, or is malformed. See the screenshot under https://github.com/artis3n/ansible-role-tailscale#insecurely_log_authkey demonstrating the error message that appears if the API key is not valid for the Tailscale account.

The ignored error on `Get Existing State Checksum` is expected, and part of the idempotence checks.

**Comment by artis3n** [comment]
This is fixed in `3.2.1` of this role, now published on Ansible Galaxy. @panos-- I really like the additional changes you have in your PR, so looking forward to merging that as well after a few review comments.
